packages/web/src/components/spa/HackAUBG/LandingAnimation/landing_animation.css

```diff
@@ -33,28 +33,17 @@
 }
 
 .hackaubg-landing-content p {
-    font-size: 1rem;
+    font-size: 1.2rem;
     margin: 4% 0 6% 0;
     width: 90%;
     align-items: center;
     justify-content: center;
 }
 
-@media only screen and (max-width: 1300px) {
-    .hackaubg-landing-content h1 {
-        font-size: 3.8rem;
-    }
-    .hackaubg-landing-content p {
-        font-size: 0.9rem;
-        padding: 2% 0 2% 0;
-        text-overflow: clip;
-    }
+@media only screen and (max-width: 900px) {
     .hackaubg-landing-section-text-animation-container {
         height: 100vh;
```

Comment:
isn't it supposed to always be 100vh? why is this set in the media-query and not at a top level?